htdocs/helpFiles/Entering-Units.html

```diff
@@ -55,7 +55,7 @@ <h2 style="text-align: center">Units Available in WeBWorK</h2>
 	</tr>
 	<tr>
 		<td style="padding: 0.5rem; border: 1px solid black">Meters</td>
-		<td style="padding: 0.5rem; border: 1px solid black; text-align: center">m</td>
+		<td style="padding: 0.5rem; border: 1px solid black; text-align: center">m or meters</td>
```

Comment:
or `meter` or `metre` or `metres`

Reply:
Sorry, I thought you were adding the "re" variants here. Nevermind about that for this.